TA_Scripts/Update_Students/createNewYml.js

```diff
@@ -0,0 +1,69 @@
+/*
+  Developed by Ten Zhi-Yang, 2017
+  To run this script, run:
+  node createNewYml.js
+
+  it will generate a new students.yml in the same folder for you to manually update. (instead of replacing the original file)
+  in order to follow existing standards, it will also generate the image id in lowercase and hyphens. The images will have to be renamed manually
+
+  Disclaimer: file follows the Javascript Standard Style, so if you don't like the lack of semi-colons, too bad!
+*/
+
+// generate csv from google docs and put in the same folder, update this name
+var csvFilename = 'contactlist.csv'
```

Comment:
We should remove the CSV file and just add the headers here as reference for future batches to use.